### ISSUE TITLE
Print class name in error message for non-initialized fields of inner classes

### ISSUE THREAD

**Issue by lazaroclapp** [uber/NullAway#365]
Consider the code in a `Foo.java` file:

```
class Foo {
   @Nullable 
   private Object foo;
   ...
   private static class Bar {
      private Object foo;
      ...
   }
}
```

Currently, if `Bar.foo` is not properly initialized, the user will get an error like:

```
Foo.java:XX: error: [NullAway] @NonNull field foo not initialized
    private Object foo;
```

With only the line number `XX` giving any indication that this refers to `Bar.foo`, not `Foo.foo`. Given the file name, the user will naturally check `Foo.foo`, see it's `@Nullable` and be confused/lose confidence in the tool, even though the warning is legitimate for `Bar.foo`.

This is a rare corner case, but it is bad UX, and could be fixed by printing the inner class as well as the field name (e.g. `Bar.foo`) whenever the field is not from the outermost class in the file.

This is low priority, but would be an excellent "first issue" for someone dipping their toes in NullAway 😄

**Comment by mayankkt9** [comment]
Can I take this issue ?

**Comment by msridhar** [comment]
Sure! Let us know if you need help

**Comment by shtke** [comment]
I was already working on this Issue. Please check my PR.

**Comment by msridhar** [comment]
Sorry @shtke I didn't know.  I will re-assign to you.  @mayankkt9 would you like help finding another issue to work on?

**Comment by mayankkt9** [comment]
Sure, I will find something else :)

**Comment by lazaroclapp** [comment]
Closing this after #375 . Thanks, @shtke !
